src/App.js

```diff
@@ -56,6 +56,7 @@ function App() {
 
   return (
     <div className="container">
+      <GlobalStyle />
       <Header />
       <main className="main">
         <div className="realDecideContainer">
```

Comment:
Later, these classNames should be replaced too
```suggestion
    <div>
      <GlobalStyle />
      <Header />
      <main>
        <div>
```